1skeleton/build.gradle

```diff
@@ -17,12 +17,16 @@ subprojects {
     task copyDockerResources(type: Sync) {
         from "$rootProject.projectDir/src/main/resources/global"
         from("$rootProject.projectDir/src/main/resources/dockerfiles/") {
-            if (project.alfresco.version.major.toInteger() < 6) {
+            if(project.alfresco.version.major.toInteger() > 6) {
+                include "Dockerfile-skeleton-7"
+                rename "Dockerfile-skeleton-7", 'Dockerfile'
+            }
+            else if (project.alfresco.version.major.toInteger() < 6) {
```

Comment:
It's clearer to say `== 6` rather than this construct.
Then the structure is also descending - 7, 6, pre6. Now it's 7, pre-6, 6..